### PR TITLE
Update oauth-with-online-and-on-premises.md

### PR DESCRIPTION
**Pull request by jambirk**
@carolynblanding

added a note to this article in response to https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/609  please either approve in GitHub or leave a comment saying you approve the change. Telling me the note shouldn't be added or that we need to talk more is also okay!

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ This topic applies to integration with Exchange Server 2013 through 2019.
 
 - For information about compatibility, see [Skype for Business compatibility with Office apps](https://docs.microsoft.com/skypeforbusiness/plan-your-deployment/clients-and-devices/compatibility-with-office).
 
+> [NOTE]
+> If you have multiple primary SIP/SMTP domains on the same Exchange server. repeat  Step 3 (using a different account name), Step 4, and Step 7 for each domain.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises.md]:
```suggestion
> If you have multiple primary SIP/SMTP domains on the same Exchange server, repeat  Step 3 (using a different account name), Step 4, and Step 7 for each domain.
```

File: Skype/SfbServer/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ This topic applies to integration with Exchange Server 2013 through 2019.
 
 - For information about compatibility, see [Skype for Business compatibility with Office apps](https://docs.microsoft.com/skypeforbusiness/plan-your-deployment/clients-and-devices/compatibility-with-office).
 
+> [NOTE]
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises.md]:
```suggestion
> [!NOTE]
```

**Comment by jambirk** [comment]
@carolynblanding  do you agree? I seem to remember discussing this...

**Comment by carolynblanding** [comment]
@jambirk 	No we don't need to do this steps more then once. I have multiple sip/smtp domains in my tenant and only performed once and no issues.

**Comment by jambirk** [comment]
Closing this per Carolyns comments.

**Pull request closed without merging**